coordinator/src/services/communication/sandbox-spine-client.ts

```diff
@@ -5,24 +5,30 @@ import * as fhir from "../../models/fhir"
+        })
+      case hl7V3.Hl7InteractionIdentifier.NOMINATED_PRESCRIPTION_RELEASE_REQUEST._attributes.extension:
+        return Promise.resolve({
+          statusCode: 200,
+          // eslint-disable-next-line max-len
+          body: sandboxReleaseResponse
```

Comment:
Think it would be good to move the other spine mock responses into files like you've done for this one, doesn't need to be part of this PR though

Reply:
Ill raise a ticket :+1: